### PR TITLE
fixed landscape filter bug

### LINKED ISSUE #2045: [Bug] Landscape badge enlists all community members when selected
**Description**
The community site (https://layer5.io/community/members) shows all members of the community (Active, Inactive) but when the Landscape option is selected all members are shown instead of ones who have the badge

**Expected Behavior**
The Landscape option should display the members who have the Landscape badge and not all

**Screenshots**

image

image

---

[Optional] **To Reproduce**
1. Go to the site
2. Click on Landscape
3. See the members
4. Then click on All Members
5. Compare them

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
fixed the landscape filter

This PR fixes #2045

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/pages/community/members.js [hunk 1]
```diff
@@ -37,22 +37,22 @@ import smpIcon from "../../assets/images/service-mesh-performance/icon/smp-dark.
 import inactiveIcon from "../../assets/images/status/inactive.png";
 import activeIcon from "../../assets/images/status/active.png";
 
- const options = [
-   { label: <DropdownWrapper><option disabled className="category comm">BADGES</option></DropdownWrapper>, value: "", color: `${theme.linkColor}`,isFixed: true},
-   { label: <DropdownWrapper><option className="allOptions">Community</option></DropdownWrapper>, value: "community", color: `${theme.linkColor}`, isFixed: true, icon: `url(${communityIcon})`},
-   { label: <DropdownWrapper><option className="allOptions">Landscape</option></DropdownWrapper>, value: "lanscape", color: `${theme.linkColor}`, isFixed: true, icon: `url(${landscapeIcon})`},
```
Comment (alphaX86) [inline src/pages/community/members.js:43]: So that's the reason why it showed all members... xD

Comment (warunicorn19) [reply src/pages/community/members.js:43]: yeah😂

**Comment by l5io** [comment]
🚀 Preview for commit d5fbaebc43c6ac7e19c011a88d174240819eddb5 at: https://6104277be201df105c6bd2a8--layer5ng.netlify.app

**Review by alphaX86 — APPROVED** [review]
Works good!

**Review by leecalcote — APPROVED** [review]
Nice work, guys.

**Pull request merged**